slick.grid.js

```diff
@@ -379,7 +381,7 @@ if (typeof Slick === "undefined") {
             //.on("click", handleClick)
             .on("scroll", handleScroll);
         $headerScroller
-            //.on("scroll", handleHeaderScroll)
+            .on("scroll", handleHeaderScroll)
```

Comment:
Any idea why this was commented out? (git blame can help find out where that happened)

Reply:
I tried to find that out but it goes back to your initial commit 13 months ago

Reply:
go to the previous repo and see if you can find anything.

Reply:
https://github.com/anthonydresser/SlickGrid this is the original repo. found it by checking the history of package.json of ads.